src/core/Dates.js

```diff
@@ -0,0 +1,26 @@
+import moment from 'moment';
+
+/**
+ * Formats the date from UTC format into string
+ *
+ * @param  {object} with start and end numbers
+ * @return {string}
+ */
+const returnFormattedDate = (dates) => {
```

Comment:
You can indicate in a clearer way the variables needed, like:

Idea 1: ({ start, end })

Idea 2: (startDate, endDate)